packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -657,75 +684,86 @@ class SemanticsNode extends AbstractNode {
   }
 
   @override
-  String toString() {
-    final StringBuffer buffer = new StringBuffer();
-    buffer.write('$runtimeType($id');
-    if (_dirty)
-      buffer.write(' (${ owner != null && owner._dirtyNodes.contains(this) ? "dirty" : "STALE; owner=$owner" })');
-    if (_shouldMergeAllDescendantsIntoThisNode)
-      buffer.write(' (leaf merge)');
+  String toStringShort() => '$runtimeType#$id';
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    bool hideOwner = true;
+    if (_dirty) {
+      final bool inDirtyNodes = owner != null && owner._dirtyNodes.contains(this);
+      properties.add(new FlagProperty('inDirtyNodes', value: inDirtyNodes, ifTrue: 'dirty', ifFalse: 'stale'));
```

Comment:
please keep the word "stale" uppercase, is it is supposed to be a cry for help

Reply:
OOPS. done.